cloud-controller-manager/do/loadbalancers_test.go

```diff
@@ -85,8 +86,38 @@ func (f *fakeLBService) RemoveForwardingRules(ctx context.Context, lbID string,
 	return f.removeForwardingRulesFn(ctx, lbID, rules...)
 }
 
+func newKVLBService(store map[string]*godo.LoadBalancer) fakeLBService {
```

Comment:
Really appreciate moving towards an LB fake. 👍 

If this continues to work with the current code, then I'm all happy. FWIW, I'm working on a branch that [fakes the LB](https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/b1cb741066efa35a3ae3fdeedebfec6c350f208d/cloud-controller-manager/do/fakelbsvc_test.go) as well. It's a bit more sophisticated since I'm already covering more cases. Additionally, I found out that I had to do things like copying LB structs to ensure that modifying them in production code does not impact the fake store.

Feel free to take inspiration from my WIP branch or ignore it entirely as needed by your PR.